examples/ITK_Example20_PixelTypes.ipynb

```diff
@@ -0,0 +1,325 @@
+   "id": "69c5eeb5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ParameterObject (000002CA822664B0)\n",
+      "  RTTI typeinfo:   class elastix::ParameterObject\n",
+      "  Reference Count: 1\n",
+      "  Modified Time: 173699\n",
+      "  Debug: Off\n",
+      "  Object Name: \n",
+      "  Observers: \n",
+      "    none\n",
+      "ParameterMap 0: \n",
+      "  (CenterOfRotationPoint 31.5 31.5 31.5)\n",
+      "  (CompressResultImage \"false\")\n",
+      "  (ComputeZYX \"false\")\n",
+      "  (DefaultPixelValue 0)\n",
+      "  (Direction 1 0 0 0 1 0 0 0 1)\n",
+      "  (FinalBSplineInterpolationOrder 1)\n",
+      "  (FixedImageDimension 3)\n",
+      "  (FixedInternalImagePixelType \"float\")\n",
+      "  (HowToCombineTransforms \"Compose\")\n",
+      "  (Index 0 0 0)\n",
+      "  (InitialTransformParametersFileName \"NoInitialTransform\")\n",
+      "  (MovingImageDimension 3)\n",
+      "  (MovingInternalImagePixelType \"float\")\n",
+      "  (NumberOfParameters 6)\n",
+      "  (Origin 0 0 0)\n",
+      "  (ResampleInterpolator \"FinalBSplineInterpolator\")\n",
+      "  (Resampler \"DefaultResampler\")\n",
+      "  (ResultImageFormat \"nii\")\n",
+      "  (ResultImagePixelType \"double\")\n",
+      "  (Size 64 64 64)\n",
+      "  (Spacing 1 1 1)\n",
+      "  (Transform \"EulerTransform\")\n",
+      "  (TransformParameters 0 0 0 0 0 0)\n",
+      "  (UseDirectionCosines \"true\")\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Copy the previous transformation result to two new parameter objects\n",
+    "transform_parameters_float = itk.ParameterObject.New()\n",
+    "transform_parameters_short = itk.ParameterObject.New()\n",
+    "\n",
+    "transform_parameters_float.AddParameterMap(result_transform_parameters.GetParameterMap(0))\n",
+    "transform_parameters_short.AddParameterMap(result_transform_parameters.GetParameterMap(0))\n",
+    "\n",
+    "# Switch to linear interpolation for resampling\n",
+    "transform_parameters_float.SetParameter(0, 'FinalBSplineInterpolationOrder', '1')\n",
+    "transform_parameters_short.SetParameter(0, 'FinalBSplineInterpolationOrder', '1')\n",
+    "\n",
+    "# Set transform to zero so that the input image matches the output image\n",
+    "transform_parameters_float.SetParameter(0, 'TransformParameters', 6 * ['0']) # 6 transformation parameters to zero\n",
+    "transform_parameters_short.SetParameter(0, 'TransformParameters', 6 * ['0'])\n",
+    "\n",
+    "print(transform_parameters_float)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7d610c09",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "For float internal pixel type - Diff min: 0.0 | max: 0.0\n",
+      "For short internal pixel type - Diff min: -0.9999923706054688 | max: -6.811788807681296e-06\n"
```

Comment:
I'm surprised that the Diff min `-0.9999923706054688` is _so_ close to -1. But I guess that's just because of the numbers generated by numpy random number generator.

Reply:
It is not that unlikely beaue we are drawing 64*64*64=262144 samples between [0, 1]. I let ChatGPT do the calculation, and it appears that there is 99.27% chance at least one number is >0.99999.